src/bp/ui-studio/src/web/views/OneFlow/diagram/nodes/Block/index.tsx

```diff
@@ -267,6 +269,7 @@ export class BlockWidgetFactory extends AbstractNodeFactory {
   private getExpandedNodes: () => string[]
   private setExpandedNodes: (id: string, expanded: boolean) => void
   private getDebugInfo: (nodeName: string) => NodeDebugInfo
+  private isSelected: (nodeId: string) => boolean
```

Comment:
Technically the node is not "selected", it's being highlighted (since only one can be selected at a time)